diff --git a/fit/journey-smoke.fit.spec.js b/fit/journey-smoke.fit.spec.js
index 3806207b..c707fb09 100644
--- a/fit/journey-smoke.fit.spec.js
+++ b/fit/journey-smoke.fit.spec.js
@@ -157,7 +157,7 @@ test.describe('live-animals journey glue', () => {
     await completeAnswerSections(page)
 
     const [document] = values.documents
-    await page.getByRole('link', { name: 'Uploaded documents' }).click()
+    await page.getByRole('link', { name: 'Upload documents' }).click()
     await addDocument(page, document)
     await expect(
       page.locator('.govuk-table__row', {
diff --git a/fit/live-animals-journey.js b/fit/live-animals-journey.js
index e92d1305..a8e83d1d 100644
--- a/fit/live-animals-journey.js
+++ b/fit/live-animals-journey.js
@@ -427,12 +427,12 @@ export const completeAnswerSections = async (
   await overview()
 
   if (!skipAnimalIdentification) {
-    await task('Animal identification details')
+    await task('Identification details')
     await answerAnimalIdentification(page)
     await overview()
   }
 
-  await task('Main reason for importing')
+  await task('Main reason for import')
   await answerImportReason(page)
   await answerAdditionalDetails(page)
 
@@ -445,7 +445,7 @@ export const completeAnswerSections = async (
   await answerTransitCountries(page)
   await answerTransporter(page)
 
-  await task('Contact address')
+  await task('Contact address for this consignment')
   await answerContactAddress(page)
 }
 
